README.md

```diff
@@ -14,15 +14,62 @@
 <div align="center">
   <h3>📥 Download Latest Release</h3>
   
-  [![macOS Download](https://img.shields.io/badge/macOS-Download-blue.svg)](https://github.com/platinum-hill/cobolt/releases/latest/download/Cobolt.dmg)
```

Comment:
Isn't latest better, and we kind of manage the .dmg page accordingly

Can be fixed later also.